indrel/index.html

```diff
@@ -2,6 +2,7 @@
 layout: default
 title: Industrial Relations
 section_id: indrel
+cover-pic: /images/group/imc-event-story-larger-bg.jpeg
```

Comment:
I think "imc-event-story.jpg" would be a better choice here for consistency since "imc-event-story-larger-bg.jpeg" is surrounded by a white border which doesn't show up for the other pages.

Reply:
I put it on a white background because it seemed to fit better. 
larger-bg: 
![image](https://user-images.githubusercontent.com/90210284/199853766-c4ab994c-93b0-46f3-895c-6c821ae1160a.png)
regular image: 
![image](https://user-images.githubusercontent.com/90210284/199853812-2d7f9ea5-47a6-4796-80d8-186074480fad.png)
Should we go with the regular image anyway?

Reply:
I think regular for now just so that it fits with the professional services page better (fills the entire horizontal space). We can definitely consider your idea of the white background in the future.